content/blog/2023/2023-04-23-Java-9-module-system.md

```diff
@@ -0,0 +1,169 @@
+# Java 9 Module System
+
+Table of Contents
```

Comment:
Please remove the table of contents, that is auto-generated by the blog from the headers.